.github/actions/updatecli/action.yml

```diff
@@ -43,6 +43,11 @@ runs:
   using: "composite"
   steps:
 
+    - uses: actions/github-script@v7
+      with:
+        script: |
+          core.warning('This action is deprecated. Use `updatecli/updatecli-action` with GitHub secrets.')
```

Comment:
What about https://github.com/elastic/oblt-actions/tree/main/updatecli/run ?

Reply:
good catch! I missed it

Reply:
```suggestion
          core.warning('This action is deprecated. Use https://github.com/elastic/oblt-actions/tree/main/updatecli/run.')
```